nodes/mixnode/src/main.rs

```diff
@@ -9,23 +17,34 @@ struct Args {
     config: std::path::PathBuf,
 }
 
+#[derive(Deserialize, Debug, Clone, Serialize)]
+pub struct Config {
+    // pub log: <Logger as ServiceData>::Settings,
+    pub mixnode: <MixnetNodeService as ServiceData>::Settings,
+}
+
+#[derive(Services)]
+struct Mixnetnode {
+    node: ServiceHandle<MixnetNodeService>,
+}
```

Comment:
I'm getting this error. I'm taking a look.
```
thread 'main' panicked at 'Cannot start a runtime from within a runtime. This happens because a function (like `block_on`) attempted to block the current thread while the thread is being used to drive asynchronous tasks.', /Users/yjlee/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.29.1/src/runtime/scheduler/multi_thread/mod.rs:86:9
``

Reply:
I fixed it: b3c96eb3ac795697530efcfa0632ca28666b4dc4